pkg/services/password/password.go

```diff
@@ -0,0 +1,58 @@
+package password
```

Comment:
This is not really a service and I don't see a case where it should be. Probably put this in the util package?

Reply:
There are a couple of things to consider for this decision.

1. The password validation was done at the user model. Decoupling from that adds flexibility in the future. Moving this static function to utils will achieve the same goal.
2. The password validation is hard-coded. But creating a service for this gives us room for future improvement.

These 2 points were the main reason I decided to use a new service, rather than adding it to utils.

@kalleep do you have a strong preference to move this to the `utils` package?

Reply:
As suggested by @kalleep I'll be moving this to its own package under the `user` package.